cluster-autoscaler/config/const.go

```diff
@@ -32,4 +36,53 @@ const (
 	DefaultScaleDownUnreadyTimeKey = "scaledownunreadytime"
 	// DefaultMaxNodeProvisionTimeKey identifies MaxNodeProvisionTime autoscaling option
 	DefaultMaxNodeProvisionTimeKey = "maxnodeprovisiontime"
+
+	// Custom scheduler configs for testing
```

Comment:
I guess putting test consts in `const.go` feels a little out of place. But I don't have a strong opinion on where this should go. I will take another look at this (P.S.: open to suggestions).

Reply:
I have moved the test config here to a separate package under at `cluster-autoscaler/config/test/config.go`
This test config is used at two different places: 
1. simulator/predicatechecker/schedulerbased_test.go
2. utils/scheduler/scheduler_test.go

Other place this can go into is the existing `cluster-autoscaler/utils/test/test_utils.go`. Since this is config for testing, putting it under a separate `test` package under `cluster-autoscaler/config` seems like a better place. 

Let me know what you think.